integrated_channels/integrated_channel/models.py

```diff
@@ -401,7 +401,28 @@ class ContentMetadataItemTransmission(TimeStampedModel):
         null=True,
     )
     deleted_at = models.DateTimeField(
```

Comment:
Nit: consider using a prefix like `remote_` for these three columns, to make it extra super-duper clear that it's talking about the CUD remote actions and when they occurred. (Might be too much of a PITA because you'd have to rename the existing `deleted_at` column).

Reply:
good idea. i am going to generate the migration last and I will tackle a column rename in that step. shouldnt be a big deal - i'll tweak the code